src/oncall/ui/static/js/oncall.js

```diff
@@ -63,7 +70,11 @@ var oncall = {
     this.defineRoutes();
     this.events.call(this);
     this.registerHandlebarHelpers();
-    this.modal.init(this);
+    this.getIrisSettings();
+    this.data.irisSettingsPromise.done(function() {
+      self.modal.init(self);
+    });
+    this.multiSelect.init();
```

Comment:
I think you have a merge conflict here. I moved the multiselect out of the global init and moved it to notifications page since that's the only place it's used